flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/codegen/calls/BuiltInMethodsTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.planner.codegen.calls;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+/** This test class is checking if methods defined in {@link BuiltInMethods} are valid. */
+class BuiltInMethodsTest {
+
+    private static Stream<Method> testMethodsAreAvailable() {
+        return Arrays.stream(BuiltInMethods.class.getMethods())
+                .filter(
+                        m ->
+                                Modifier.isStatic(m.getModifiers())
+                                        && Modifier.isPublic(m.getModifiers()));
+    }
+
+    @ParameterizedTest
+    @MethodSource
+    void testMethodsAreAvailable(Method m) throws Exception {
+        assertDoesNotThrow(
```

Comment:
Maybe add a comment regarding this invocation with `null` and the specific use case on static methods.